pkgs/servers/mautrix-telegram/default.nix

```diff
@@ -1,18 +1,28 @@
 { lib, python3, mautrix-telegram, fetchFromGitHub }:
 
-with python3.pkgs;
-
 let
+  python = python3.override {
+    packageOverrides = self: super: {
+      sqlalchemy = super.sqlalchemy.overridePythonAttrs (oldAttrs: rec {
```

Comment:
```suggestion
    packageOverrides = final: prev: {
      sqlalchemy = prev.sqlalchemy.overridePythonAttrs (oldAttrs: rec {
```

Reply:
Come on, the other one is way more common.